enterprise/admin/views.py

```diff
@@ -229,28 +232,47 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request):
             course_id: The ID of the course in which we want to enroll
             mode: The enrollment mode the users will be enrolled in the course with
             request: The HTTP request the enrollment is being created by
+            notify: Whether to notify (by email) the users that have been enrolled
         """
         enrolled = []
         non_existing = []
         failed = []
         enrollment_client = EnrollmentApiClient()
+        course_details = CourseApiClient().get_course_details(course_id)
+        reverse_method = get_reverse_by_site(request, enterprise_customer.site)
+        course_url = reverse_method('about_course', args=(course_id,))
+        email_conn = mail.get_connection()
+        email_conn.open()
```

Comment:
Good catch! From what I can tell, though, those two items are mutually exclusive by default, so I'm putting together a context manager that conditionally opens and closes a connection.